stubs/setuptools/setuptools/modified.pyi

```diff
@@ -0,0 +1,8 @@
+from ._distutils._modified import (
+    newer as newer,
+    newer_group as newer_group,
+    newer_pairwise as newer_pairwise,
+    newer_pairwise_group as newer_pairwise_group,
+)
```

Comment:
I could define the methods here instead of in `setuptools/_distutils/_modified` to avoid a adding a private module

Reply:
Better to define them where they're defined at runtime imo, as you currently have it